sktime/base/_base.py

```diff
@@ -565,46 +565,88 @@ def get_fitted_params(self, deep=True):
                 "fitted yet, please call fit on data before get_fitted_params"
             )
 
-        # collect non-nested fitted params of self
-        fitted_params = self._get_fitted_params()
```

Comment:
I think removing this is a mistake, you are missing the root node?

Reply:
I've handled this case when `deep=False`:
``` python
if not deep:
    return self._gfp_non_nested_plugin()
```

In cases where `deep=True`, all the plugins are called in the for loop (including the non-nested fitted params, which is essentially a call to `_get_fitted_params()`)

Reply:
I see, yes, indeed.